packages/assets-controllers/src/AccountTrackerController.ts

```diff
@@ -309,29 +303,6 @@ export class AccountTrackerController extends StaticIntervalPollingController<Ac
     };
   }
 
-  /**
-   * Starts a new polling interval.
-   *
-   * @param interval - Polling interval trigger a 'refresh'.
-   */
-  async poll(interval?: number): Promise<void> {
```

Comment:
Nice, so we are removing this old logic in favour of the methods provided by the `StaticIntervalPollingController`.

Reply:
yes , the interface already provide startPolling function